tests/next_tests/integration_tests/feature_tests/ffront_tests/test_sparse_field.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2023, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from gt4py.next.ffront.decorator import field_operator, program
+from gt4py.next.ffront.fbuiltins import Field, int64, neighbor_sum
+
+from next_tests.integration_tests.feature_tests.ffront_tests.ffront_test_utils import *
+
+
+def test_sparse_field(reduction_setup, fieldview_backend):
```

Comment:
There are already tests in `test_gt4py_builtins.py`, namely `test_reduction_execution_nb`, that cover exactly this case. I would remove this one here and enable all of them (they are currently disabled).

Reply:
These tests are illegal because they are operating on only a sparse field (and for no reason). I think they should be changed to operate on shifted fields.